<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataQuery</class>
 <widget class="QWidget" name="DataQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1117</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,2,2,2,1,1">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>起始日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="startdt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>终止日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="enddt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="factorname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="deviceaddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="query">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1,1">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="ck_rtk">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(0, 0, 0, 0);
	color:black;
    border: 0px solid #b1b1b1;
	width: 30px;
	height: 32px;
}
QCheckBox:enabled:checked{
        color: rgb(10, 160, 0);
}
QCheckBox:enabled:!checked{
        color: rgb(0, 0, 0);

}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
    image:url(&quot;:/new/images/image/Off.png&quot;)
}

QCheckBox::indicator:checked {
    image:url(&quot;:/new/images/image/On.png&quot;)
}
</string>
          </property>
          <property name="text">
           <string>实时数据</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_min">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(0, 0, 0, 0);
	color:black;
    border: 0px solid #b1b1b1;
	width: 30px;
	height: 32px;
}
QCheckBox:enabled:checked{
        color: rgb(10, 160, 0);
}
QCheckBox:enabled:!checked{
        color: rgb(0, 0, 0);

}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
    image:url(&quot;:/new/images/image/Off.png&quot;)
}

QCheckBox::indicator:checked {
    image:url(&quot;:/new/images/image/On.png&quot;)
}
</string>
          </property>
          <property name="text">
           <string>分钟数据</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_hour">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(0, 0, 0, 0);
	color:black;
    border: 0px solid #b1b1b1;
	width: 30px;
	height: 32px;
}
QCheckBox:enabled:checked{
        color: rgb(10, 160, 0);
}
QCheckBox:enabled:!checked{
        color: rgb(0, 0, 0);

}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
    image:url(&quot;:/new/images/image/Off.png&quot;)
}

QCheckBox::indicator:checked {
    image:url(&quot;:/new/images/image/On.png&quot;)
}
</string>
          </property>
          <property name="text">
           <string>小时数据</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_day">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(0, 0, 0, 0);
	color:black;
    border: 0px solid #b1b1b1;
	width: 30px;
	height: 32px;
}
QCheckBox:enabled:checked{
        color: rgb(10, 160, 0);
}
QCheckBox:enabled:!checked{
        color: rgb(0, 0, 0);

}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
    image:url(&quot;:/new/images/image/Off.png&quot;)
}

QCheckBox::indicator:checked {
    image:url(&quot;:/new/images/image/On.png&quot;)
}
</string>
          </property>
          <property name="text">
           <string>日数据</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_month">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(0, 0, 0, 0);
	color:black;
    border: 0px solid #b1b1b1;
	width: 30px;
	height: 32px;
}
QCheckBox:enabled:checked{
        color: rgb(10, 160, 0);
}
QCheckBox:enabled:!checked{
        color: rgb(0, 0, 0);

}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
    image:url(&quot;:/new/images/image/Off.png&quot;)
}

QCheckBox::indicator:checked {
    image:url(&quot;:/new/images/image/On.png&quot;)
}
</string>
          </property>
          <property name="text">
           <string>月数据</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_trans">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(0, 0, 0, 0);
	color:black;
    border: 0px solid #b1b1b1;
	width: 30px;
	height: 32px;
}
QCheckBox:enabled:checked{
        color: rgb(10, 160, 0);
}
QCheckBox:enabled:!checked{
        color: rgb(0, 0, 0);

}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
    image:url(&quot;:/new/images/image/Off.png&quot;)
}

QCheckBox::indicator:checked {
    image:url(&quot;:/new/images/image/On.png&quot;)
}
</string>
          </property>
          <property name="text">
           <string>传输报文</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="tableStack">
       <widget class="QWidget" name="pagertk">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tablertk"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pagemin">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tablemin"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pagehour">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tablehour"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageday">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableday"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pagemonth">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tablemonth"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pagetrans">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tabletrans"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
